packages/config/types/build.d.ts

```diff
@@ -28,6 +28,7 @@ interface NuxtBabelOptions extends Pick<TransformOptions, Exclude<keyof Transfor
 
 export interface NuxtConfigurationBuild {
   analyze?: BundleAnalyzerPlugin.Options | boolean
+  indicator?: boolean
```

Comment:
Move `indicator` below (between `html` & `loaders`) so it fits alphabetical consistency

Reply:
Fixed, thanks Kevin, I don't know how I missed this!